File: java/src/test/java/io/amerhwitat/bizx/UnifiedBizXRuntimeTest.java
package io.amerhwitat.bizx;

import static org.junit.jupiter.api.Assertions.*;
import org.junit.jupiter.api.Test;

class UnifiedBizXRuntimeTest {
    @Test void exposesAllFeatureFamilies() {
        var r = new UnifiedBizXRuntime();
        assertTrue(r.features().contains("game"));
        assertTrue(r.features().contains("NetworkUnified"));
        assertTrue(r.features().contains("crypto"));
        assertTrue(r.features().contains("3D"));
        assertTrue(r.features().contains("game-store"));
    }

    @Test void sha256IsDeterministic() {
        assertEquals("d5de73ee1d25f202d90d5a3e8c6eb4e90b1d23ecfad9a9b828c8a6ed8f322b18", new UnifiedBizXRuntime().sha256("BizX-test-placeholder"));
    }
}

File: kotlin/src/test/kotlin/io/amerhwitat/bizx/UnifiedBizXRuntimeTest.kt
package io.amerhwitat.bizx

import kotlin.test.Test
import kotlin.test.assertTrue

class UnifiedBizXRuntimeTest {
    @Test fun exposesFeatureParity() {
        val features = UnifiedBizXRuntime().features
        assertTrue("game" in features)
        assertTrue("NetworkUnified" in features)
        assertTrue("crypto" in features)
        assertTrue("3D" in features)
    }
}
